editor/src/core/workers/ts/ts-worker.spec.ts

```diff
@@ -9,16 +9,18 @@ import {
 } from '../../shared/project-file-types'
 import { convertScenesToUtopiaCanvasComponent } from '../../model/scene-utils'
 
-import * as SampleTypeDefinitions from './sample-type-definitions.json'
+import SampleTypeDefinitions from './sample-type-definitions.json'
 import { contentsToTree } from '../../../components/assets'
 import { emptyComments } from '../parser-printer/parser-printer-comments'
 import { jsxAttributesFromMap } from '../../shared/element-template'
 
 describe('Typescript worker builds the project', () => {
   it('initializing a new project', (done) => {
     handleMessage(SampleInitTSWorkerMessage, (msg) => {
-      expect(msg).toMatchSnapshot()
-      done()
+      if (msg.type === 'build') {
+        expect(msg).toMatchSnapshot()
+        done()
+      }
```

Comment:
Is there now a risk of this never completing?

Reply:
nope, Jest has a global timeout on tests (currently set to 10000 ms)